### DISCUSSION TITLE
React's official response, regarding the chaos in issue managers of React repos, is missing.

### DISCUSSION THREAD

**Discussion by ShikiSuen** [react-native-community/discussions-and-proposals#468] [Ideas]
Mr. @cortinico indicated the existence of "The official response from the React Team" in this URL which is no longer accessible anymore:
https://github.com/facebook/react/issues/23402#issuecomment-1058227670

Could you please provide a new link to this statement? At least I had no chance to read that.

==================
Update: @gaearon posted the reply again. In case that the thread may gets removed, please forgive my decision of posting the screenshot here as a backup (for references purposes).
<img width="930" alt="image" src="https://user-images.githubusercontent.com/3164826/157692959-8bbc2171-c1ba-4196-9e4a-c32d606359eb.png">

**Comment by cortinico** [comment]
Thanks for pointing this out @ShikiSuen. For some reason, the post I linked got deleted. [You can find a repost of the same message here](https://github.com/reactjs/reactjs.org/issues/4433#issuecomment-1060759889)

**Reply by ShikiSuen** [reply]
Thank you for the response.
